internal/coremsgs/en_config_descriptions.go

```diff
@@ -20,301 +20,268 @@ import (
 	"github.com/hyperledger/firefly/pkg/i18n"
 )
 
-var ffm = i18n.FFM
+var ffc = i18n.FFC
+var timeDurationType = "[`time.Duration`](https://pkg.go.dev/time#Duration)"
```

Comment:
Would be nice for these to be exposed, so I could re-use them in docs for a Microservice using this config utility

Reply:
That makes sense. I can probably move them somewhere that would do that

Reply:
Thanks @nguyer 👍